server/manager/src/main/java/org/apache/accumulo/manager/ManagerClientServiceHandler.java

```diff
@@ -555,10 +555,13 @@ private void alterTableProperty(TCredentials c, String tableName, String propert
     }
 
     try {
-      if (value == null || value.isEmpty()) {
+      if (op == TableOperation.REMOVE_PROPERTY) {
         PropUtil.removeProperties(manager.getContext(),
             TablePropKey.of(manager.getContext(), tableId), List.of(property));
-      } else {
+      } else if (op == TableOperation.SET_PROPERTY) {
+        if (value == null || value.isEmpty()) {
```

Comment:
It probably shouldn't be possible for most properties to set it to a non-empty string of whitespace. So, `value.isBlank()` might be better. However, I don't want to make assumptions about what custom properties or SPI opts properties might do, so probably best to leave it alone. This is good.